submissions/ASTkach/popup/index.html

```diff
@@ -0,0 +1,159 @@
+                                                </a>
+                                            </li>
+                                            <li class="nav__sub-item nav__sub-item--more">
+                                                <a class="nav__sub-link" href="#" target="_blank" rel="nofollow noopener" tabindex="20">
+                                                    <img src="img/icons/collection.png" alt="" class="nav__img" width="737" height="919" />
+                                                    <p class="nav__text">Collection</p>
+                                                </a>
+                                            </li>
+                                            <li class="nav__sub-item nav__sub-item--more">
+                                                <a class="nav__sub-link" href="#" target="_blank" rel="nofollow noopener" tabindex="21">
+                                                    <img src="img/icons/mars.png" alt="" class="nav__img" width="886" height="885" />
+                                                    <p class="nav__text">Mars</p>
+                                                </a>
+                                            </li>
+                                            <li class="nav__sub-item nav__sub-item--more">
+                                                <a class="nav__sub-link" href="#" target="_blank" rel="nofollow noopener" tabindex="22">
+                                                    <img src="img/icons/blogs.png" alt="" class="nav__img" width="798" height="709" />
+                                                    <p class="nav__text">Blogs</p>
+                                                </a>
+                                            </li>
+                                            <li class="nav__sub-item nav__sub-item--more">
+                                                <a class="nav__sub-link" href="#" target="_blank" rel="nofollow noopener" tabindex="23">
+                                                    <img src="img/icons/promo.png" alt="" class="nav__img" width="883" height="883" />
+                                                    <p class="nav__text">Promos</p>
+                                                </a>
+                                            </li>
+                                        </ul>
+                                    </div>
+                                </div>
+                            </li>
+                        </ul>
+                    </nav>
+                    <div class="header__user">
+                        <a href="#" class="header__notification _icon-bell" target="_blank" rel="nofollow noopener" tabindex="24"></a>
+                        <a href="#" class="header__profile _icon-logo" target="_blank" rel="nofollow noopener" tabindex="25">
+                            <!-- <img src="img/logo/logo.svg" alt="profile" class="header__icon" width="24" height="24" /> -->
```

Comment:
Avoid commented code in PR

Reply:
Already updated. Sorry.

Reply:
Did you forget to update the current PR?)

Reply:
omg... push wrong index.html ((( One minute.

Reply:
done